### PR TITLE
Readme and Sample changes.

### PR DESCRIPTION
**Pull request by Om83**
1- Updated Readme to use the new redirect URI scheme. Some more changes to make instructions clear.
2- After clicking SignOut the sample app continued to use the previous signed in account because of cookies, so had to specify the prompt type.
3- pod install was giving an error. Had to run `pod install --repo-update`. Not sure if the change in the Podfile.lock is needed.

### CODE REVIEW

File: MSALiOS/ViewController.swift [hunk 1]
```diff
@@ -33,7 +33,6 @@ import MSAL
 class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate {
     
     // Update the below to your client ID you received in the portal. The below is for running the demo only
-    let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
```
Comment (oldalton) [inline MSALiOS/ViewController.swift]: why this got removed?

Comment (Om83) [reply MSALiOS/ViewController.swift]: That was accidental. Thanks for catching.

File: README.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ urlFragment: active-directory-ios-swift-native-v2
 
 ![Build Badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/523/badge)
 
-| [Getting Started](https://docs.microsoft.com/azure/active-directory/develop/guidedsetups/active-directory-ios)| [Library](https://github.com/AzureAD/microsoft-authentication-library-for-objc) | [API Reference](https://azuread.github.io/docs/objc/) | [Support](README.md#community-help-and-support)
+| [Getting Started](https://docs.microsoft.com/azure/active-directory/develop/guidedsetups/active-directory-ios)| [Library](https://github.com/AzureAD/microsoft-authentication-library-for-objc) | [API Reference](https://azuread.github.io/docs/objc/) | [Support](README.md#feedback,community-help,-and-support)
```
Comment (jasoncoolmax) [inline README.md]: Same issue as the other PR :)

File: README.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ urlFragment: active-directory-ios-swift-native-v2
...
 | --- | --- | --- | --- |
 
-The MSAL library for iOS gives your app the ability to begin using the [Microsoft identity platform](https://aka.ms/aaddev) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com/) in a converged experience using industry standard OAuth2 and OpenID Connect. This sample demonstrates all the normal lifecycles your application should experience, including:
+The MSAL library for iOS and macOS gives your app the ability to begin using the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. This sample demonstrates all the normal lifecycles your application should experience, including:
```
Comment (jasoncoolmax) [inline README.md]: Seems `https://cloud.microsoft.com` is not reachable for me :P

File: README.md [hunk 4]
```diff
@@ -56,99 +58,79 @@ $ pod install
...
 
-In the `ViewControler.swift` file, update the `kClientID` variable with your client ID.
+In the `ViewControler.swift` file, update the `kClientID` variable with your Application (client) ID.
 
 ```swift
-    // Update the below to your client ID you received in the portal. The below is for running the demo only
+    // Update the below with the client ID you received in the portal. 
     
-    let kClientID = "<your-client-id-here>"
+    let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
```
Comment (jasoncoolmax) [inline README.md]: Just curious, what is the reason by replacing `<your-client-id-here>` with real client id here? :)

Comment (Om83) [reply README.md]: This is to make it consistent with the Mac sample and also to make sure that it works out of box. Comment above this line asks to replace if the user registers the app.

File: README.md (hunk 3 above)
Comment (oldalton) [inline README.md]: I think previous readme pointing to Microsoft identity platform was better. Do you mind reverting the change @Om83?

File: README.md [hunk 5]
```diff
@@ -56,99 +58,79 @@ $ pod install
...
-3. After the app is created, you'll land on your app management page. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app.bundle.id>://auth`. Replace <app.bundle.id> with the **Bundle Identifier** for your application.
-4. Hit the **Save** button in the top left, to save these updates. 
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Azure Active Directory** blade, and then select **App registrations**.
+3. Click on the **New registration** button at the top left of the page.
+4. On the app registration page,
+   - Name your app
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Click **Register** to finish.
+5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1C below.
```
Comment (oldalton) [inline README.md]: What is step 1C? Should it be removed from instructions?

File: README.md [hunk 6]
```diff
@@ -56,99 +58,79 @@ $ pod install
...
-            <key>CFBundleTypeRole</key>
-            <string>Editor</string>
-            <key>CFBundleURLName</key>
-            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-            <key>CFBundleURLSchemes</key>
-            <array>
-                <string>msauth.<app.bundle.id></string>
-            </array>
-        </dict>
+      <string>msauth.com.microsoft.identitysample.MSALiOS</string>
```
Comment (oldalton) [inline README.md]: should this be msauth.<app.bundle.id>?

File: README.md [hunk 7]
```diff
@@ -56,99 +58,79 @@ $ pod install
...
     </array>
+  </dict>
+</array>
 ```
 
 2. Configure your application defaults
```
Comment (oldalton) [inline README.md]: should this one be 2C?

File: README.md (hunk 4 above)
Comment (oldalton) [reply README.md]: Should we still mention that "The below is for running the demo only" for clarity?

File: README.md [hunk 8]
```diff
@@ -56,99 +58,79 @@ $ pod install
...
 
-To provide a recommendation, visit 
-our [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory).
+To provide a recommendation, visit our [User Voice page](https://feedback.azure.com/forums/169401-azure-active-directory).
 
 ## Contribute
 
-We enthusiastically welcome contributions and feedback. You can clone the repo and start 
-contributing now. Read our [Contribution Guide](Contributing.md) for more information.
+We enthusiastically welcome contributions and feedback. You can clone the repo and start contributing now. Read our [Contribution Guide](Contributing.md) for more information.
```
Comment (oldalton) [inline README.md]: Do we have contribution guide? I think we didn't have it for MSAL main repository so we removed reference to it from the readme.

File: README.md (hunk 6 above)
Comment (Om83) [reply README.md]: This is based on the instruction above.

File: README.md (hunk 8 above)
Comment (Om83) [reply README.md]: Good catch removed.

File: README.md (hunk 5 above)
Comment (Om83) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (Om83) [reply README.md]: Done.

Comment (Om83) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (Om83) [reply README.md]: Added.

File: README.md (hunk 5 above)
Comment (Om83) [reply README.md]: Fixed. It should be 2B instead.

File: README.md (hunk 6 above)
Comment (oldalton) [reply README.md]: Instructions above suggest replacing this value with msauth.[app_bundle_id]. We are showing this value in all Info.plist samples and quoickstarts. I'd recommend to follow the same pattern here.

Comment (Om83) [reply README.md]: Done.

**Pull request merged**